docs/reference/indices/refresh.asciidoc

```diff
@@ -1,28 +1,86 @@
 [[indices-refresh]]
-=== Refresh
+=== Refresh API
+++++
+<titleabbrev>Refresh</titleabbrev>
+++++
 
-The refresh API allows to explicitly refresh one or more index, making
-all operations performed since the last refresh available for search.
-The (near) real-time capabilities depend on the index engine used. For
-example, the internal one requires refresh to be called, but by default a
-refresh is scheduled periodically.
+Refreshes one or more indices.
 
 [source,console]
---------------------------------------------------
+----
 POST /twitter/_refresh
---------------------------------------------------
+----
 // TEST[setup:twitter]
 
-[float]
-==== Multi Index
 
-The refresh API can be applied to more than one index with a single
-call, or even on `_all` the indices.
+[[refresh-api-request]]
+==== {api-request-title}
+
+`POST <index>/_refresh`
+
+`GET <index>/_refresh`
+
+`POST /_refresh`
+
+`GET /_refresh`
+
+
+[[refresh-api-desc]]
+==== {api-description-title}
+
+Use the refresh API to explicitly refresh one or more indices.
+A *refresh* makes all operations performed on an index
+since the last refresh
+available for search.
+
+// tag::refresh-interval-default[]
+By default, Elasticsearch refreshes indices every second, but only on
+indices that have received one search request or more in the last 30 seconds.
+// end::refresh-interval-default[]
+You can change this default interval
+using the <<index-refresh-interval-setting,`index.refresh_interval`>> setting.
+
+
```

Comment:
I'd like to clarify the following points as well, which is something that users get sometimes confused about: Refreshes are a costly operation and we would recommend avoiding running them more than once per second for good performance of the cluster. As a consequence, it is a much better idea to rely on Elasticsearch's periodic refreshes than to explicitly refresh on every index operation or every search operation. If you have an application whose workflow depends on indexing a document in Elasticsearch and then running a search that _has to_ see this new document, then a better option is to wait for the periodic refresh to happen on the application before running the search, which can be done via the `refresh=wait_for` option of the index API.

Reply:
Thanks for your feedback @jpountz.

I've added an important section that covers these points with a44328ac58e744a4e36ee50ab4ce85245444b6b7.